projects/plugins/jetpack/extensions/blocks/latex-formula/edit.tsx

```diff
@@ -0,0 +1,167 @@
+		katex?: {
+			renderToString: (
+				latex: string,
+				options: { throwOnError: boolean; displayMode: boolean }
+			) => string;
+		};
+	}
+}
+
+function ensureKatexLoaded( onLoad: () => void ): void {
+	if ( typeof window.katex !== 'undefined' ) {
+		onLoad();
+		return;
+	}
+	if ( ! document.querySelector( 'link[data-katex]' ) ) {
+		const link = document.createElement( 'link' );
+		link.rel = 'stylesheet';
+		link.href = KATEX_CSS;
+		link.setAttribute( 'data-katex', 'true' );
+		document.head.appendChild( link );
+	}
+	if ( ! document.querySelector( 'script[data-katex]' ) ) {
+		const script = document.createElement( 'script' );
+		script.src = KATEX_JS;
+		script.async = true;
+		script.setAttribute( 'data-katex', 'true' );
+		script.onload = onLoad;
+		document.body.appendChild( script );
+	} else {
+		const existingScript = document.querySelector( 'script[data-katex]' ) as HTMLScriptElement;
+		existingScript.addEventListener( 'load', onLoad, { once: true } );
+	}
+}
+
+export default function Edit( {
+	attributes,
+	setAttributes,
+	isSelected,
+}: BlockEditProps< BlockAttributes > ) {
+	const { latex = '', editing = true } = attributes;
+	const [ preview, setPreview ] = useState< string >( '' );
+	const [ currentLatex, setCurrentLatex ] = useState< string >( latex );
+	const [ katexReady, setKatexReady ] = useState< boolean >( typeof window.katex !== 'undefined' );
+	const textareaRef = useRef< HTMLTextAreaElement >();
+	const props = useBlockProps( { className: 'jetpack-latex-formula is-editing' } );
+
+	useEffect( () => {
+		ensureKatexLoaded( () => setKatexReady( true ) );
+	}, [] );
+
+	useEffect( () => {
+		if ( editing && isSelected && textareaRef.current ) {
+			textareaRef.current.focus();
+		}
+	}, [ editing, isSelected ] );
+
+	useEffect( () => {
+		if ( ! katexReady ) {
+			setPreview( __( 'Loading formula preview…', 'jetpack' ) );
+			return;
+		}
+		try {
+			const html =
+				window.katex?.renderToString( currentLatex, {
+					throwOnError: false,
+					displayMode: true,
+				} ) || '';
+			setPreview( html );
+		} catch {
+			setPreview( __( 'Error rendering formula', 'jetpack' ) );
+		}
+	}, [ currentLatex, katexReady ] );
+
+	if ( editing ) {
+		return (
+			<div { ...props }>
+				<TextareaControl
+					label={ __( 'LaTeX Code', 'jetpack' ) }
+					ref={ textareaRef }
+					value={ currentLatex }
+					onChange={ ( next: string ) => setCurrentLatex( next ) }
+					help={ __( 'Enter LaTeX math to render below. Example: \\frac{a}{b}', 'jetpack' ) }
+					rows={ 4 }
+					autoFocus
+				/>
+				<div className="jetpack-latex-formula-live-preview">
+					{ katexReady ? (
+						<div
+							className="jetpack-latex-formula-rendered"
+							dangerouslySetInnerHTML={ { __html: preview } }
+						/>
+					) : (
+						<div className="jetpack-latex-formula-loading">
+							{ __( 'Loading formula preview…', 'jetpack' ) }
+						</div>
+					) }
+				</div>
+				<div className="jetpack-latex-formula-save-btn-row">
```

Comment:
[`HStack` component](https://wordpress.github.io/gutenberg/?path=/docs/components-hstack--docs) with `justify="end"` and you can get rid of the CSS:

```js
import {
	__experimentalHStack as HStack, // eslint-disable-line @wordpress/no-unsafe-wp-apis
} from '@wordpress/components';

<HStack justify="end"></HStack>
```